### ISSUE TITLE
data loader 

### ISSUE THREAD

**Issue by TaegyunKwon** [madmaclab/DCGAN_pytorch#1]
`data_loader = torch.utils.data.DataLoader(dataset=mnist,
                                                                    batch_size=100, 
                                                                    shuffle=True)'
과 같이 dataset loader를 정의할 때, train/valid/test split을 나누어서 loader를 구성할 수 있는 방법이 있나요?
custom dataset을 이용할 때에는 [예시](https://github.com/yunjey/pytorch-tutorial/blob/master/tutorials/03-advanced/deep_convolutional_gan/data_loader.py) 와 같이 __init__, __getitem__, __len__의 magic method를 가진 data.Dataset class를 만들어서 사용하는 것 같은데, 
이러한 방법으로 돌아가야 할까요?

**Comment by TaegyunKwon** [comment]
자문 자답 - 
다시 보니 
'mnist = torchvision.datasets.MNIST(root='./data/', train=True, transform=transform, download=True)`
로 mnist라는 dataset을 정의하는데, 여기서 train parameter로 train/ test 를 나누어 dataset을 만들 수 있네요
